app/templates/user/user_settings.html

```diff
@@ -13,40 +13,43 @@
 	<div class="container-fluid px-4">
 		<h1 class="mt-4">User Profile</h1>
 		<ol class="breadcrumb mb-4">
-			<li class="breadcrumb-item active">User Profile</li>
+			<li class="breadcrumb-item active px-1"> Edit User Profile</li>
 		</ol>
-		<div class="d-md-flex flex-items-center flex-justify-between mt-1 mb-4">
+		<div class="d-md-flex flex-items-center justify-content-center mt-1 mb-4">
 			<div class="d-flex flex-items-center mb-2 mb-md-0">
-				<img src="" size="48" width="48" height="48" class="avatar-user avatar mr-3">
+				<img
+                src=" https://ui-avatars.com/api/?background=random&length=1&name={{ current_user.username}}" size="48"
+					width="48" height="48" class="rounded-circle me-2">
 				<div class="flex-auto">
 					<h3 class="h3 lh-condensed"> {{ current_user.username }} </h1>
-					<div class="d-flex flex-items-center flex-wrap">
-						<p class="color-text-tertiary mb-0 mr-3"> Your personal account </p>
-					</div>
+						<div class="d-flex flex-items-center flex-wrap">
+							<a class="color-text-tertiary mb-0 mr-3" href="{{ url_for('user_dashboard.user_profile_page', username=current_user.username) }}"> Your personal account </a>
+						</div>
```

Comment:
I wanted this `a` element to be on the extreme right of this row. Try if you can make that happen, I didnt look too much into it, the code is similar to that of the GitHub profile page

Reply:
Look better this way